### PR TITLE
test: add unit/integration tests

### LINKED ISSUE #8: Write Tests

### PR DESCRIPTION
**Pull request by zacharygolba**
This PR implements Unit and Integrations tests with Travis-CI hooks to help us keep Lux stable.

*Note:*

This does not give us 100% test coverage but provides a solid foundation for tests to be run and written before we merge anything into `master`.

--

Closes #8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Lux
+# Lux [![Build Status](https://travis-ci.org/postlight/lux.svg?branch=master)](https://travis-ci.org/postlight/lux)
```
Comment (mutewinter) [inline README.md]: Also update the **Testing** section with details.

Comment (mutewinter) [inline README.md]: Would also be cool to add the badges via [Shields.io](http://shields.io). You could also get the nifty NPM badges if you wanted.

**Comment by mutewinter** [comment]
Needs a small update for the `README.md` as mentioned above.

File: README.md (hunk 1 above)
Comment (zacharygolba) [inline README.md]: Good idea! I'll make the changes this evening.

**Comment by zacharygolba** [comment]
@mutewinter NPM badge and Shields.io implementation can be done in another PR.

**Pull request merged**